modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/BindConfigurationMVCActionCommand.java

```diff
@@ -128,7 +133,47 @@ public boolean processAction(
 			properties.put(ConfigurationAdmin.SERVICE_FACTORYPID, factoryPid);
 		}
 
-		configureTargetService(configurationModel, configuration, properties);
+		try {
+			configureTargetService(
+				configurationModel, configuration, properties);
+		}
+		catch (Exception e) {
+			Throwable cause = e.getCause();
+
+			while (cause != null) {
```

Comment:
If we do proper error handling you should not need to iterate through causes. If that's not happening let's fix the error handling.